analysis/sextractor_analysis.py

```diff
@@ -1,11 +1,10 @@
+import os
 import sep
 import numpy as np
 import pylab as plt
 import pandas as pd
 
-DATADIR = "<path-to-data-dir>"
-CATDIR = "<path-to-data-dir>"
-FINALCATDIR = "<path-to-catalog-results>"
+CATDIR = "<path-to-catalog-results>"
```

Comment:
Should we do the same with that one ?
Otherwise they need to edit the file.

Reply:
Sure! I wasn't sure about the output structure of candels-blender anymore, but checking on it, seems to be the same datadir for cats?

Reply:
I've adjusted it to cat = pd.read_csv(f"{datadir}/{sample}_catalogue.csv"), now the paths/naming should fit together :)